### PR TITLE
Sufficiently slow space wind will no longer violently collide things into each other

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Ever got into a situation where someone left a hole in the floor in which a locker ended up and your character got repeatedly slammed into that locker until they died, even though the breeze wasn't all that much? You probably have!
Adds a minimum amount of airflow speed required for violent ZAS collision to happen, which prevents people and objects from ZAS-colliding into each other (they will still touch and all that though). Not much sense in being brutalized by air that gently nudges you into a locker.
I'll keep looking at this change until I'm confident I've ironed out any sort of bug from touching paleolithic code.

:cl:
 * tweak: ZAS will no longer violently collide you into objects from any sort of breeze, now it requires a sufficiently strong breeze to do so.

### CODE REVIEW

**Comment by Eneocho** [comment]
Soulless

**Comment by boy2mantwicethefam** [comment]
What is "soul"? A miserable little pile of shitcode. But enough talk... Have at you!

**Comment by Eneocho** [comment]
>What is "soul"?

Terminal c*der syndrome right here. Poor guy.

**Comment by adacovsk** [comment]
Being pinned down from a pressure difference should definitely remain a thing, but perhaps more damage on the initial contact should higher. This might also solve the server from crashing from high amounts of objects repeatedly hitting eachother

**Comment by ghost** [comment]
can one of the dissidents explain why they downboated this?

**Comment by Eneocho** [comment]
> can one of the dissidents explain why they downboated this?

I find the ZAS fuckery funny and this actively removes something I find funny. Simple as.

**Comment by adacovsk** [comment]
> > can one of the dissidents explain why they downboated this?
> 
> I find the ZAS fuckery funny and this actively removes something I find funny. Simple as.

this doesn't necessarily remove the fuckery, just the death

**Comment by djkramer123** [comment]
> can one of the dissidents explain why they downboated this?

it's funny

it being the dying part, not the act of downvoting.

**Comment by ghost** [comment]
I mean yeah it's funny, but it's not fun for the player and it doesn't make sense. If implemented properly you'd still get the funny slamming into an object 10000 times as long as there's a high pressure differential.

**Comment by aceedex** [comment]
This isn't what busta would have wanted

**Comment by Eneocho** [comment]
> I mean yeah it's funny, but it's not fun for the player and it doesn't make sense. If implemented properly you'd still get the funny slamming into an object 10000 times as long as there's a high pressure differential.

It's already implemented this way. I remember one lowpop round in packed about a year ago where I made a R-UST in the atmos room. There was a guy being a little too hostile at times so I put one of my handtele beacons in the R-UST room and warned that I would turn the station into a second sun if I were to be harmed.
Not much happened to me, but that one guy killed all but one other guy. I rushed to call it and went to guard that survivor (think it was a miner). Stuff happened in between and departures got sharted, which made me and the other survivor separate. Once the shuttle arrived the other survivor dude got onto the captain area and welded the airlock. I was just entering the shuttle when the murderman jumps in and tries to laser me. Just as he hits me I open the portal to the R-UST chamber which transports the like 300 million kPa of 1M°C plasma to the shuttle. Murderman get ZAS'd out of the shuttle and dies to slamming against the same wall a hundred times. The room was too hot n pressurized to survive so I spacewalked and made a chair on one of the windows next to the consoles. Survivor dude and I survive and we promptly suicide on centcomm. As a ghost I go check on the murderman's body and he's STILL getting ZAS'd against that wall, 3 whole minutes later.
TL;DR: It does work that way, I have experienced it in-game.

**Comment by pedr69** [comment]
what does this do in practice?
haven't been zassed into lockers and killed by them often and don't understand what you mean by lockers of death
when there isn't much airflow, enough just to blow away the pens and papers off your desk, neither you nor lockers get blown away, so i don't understand what this is supposed to accomplish

**Comment by boy2mantwicethefam** [comment]
Adds a minimum required amount of airflow strength for ZAS collision damage to actually happen. There is an occasional case where a crate or a locker can end up on a breached tile and a player gets stunlocked and killed by constantly colliding into the crate/locker while the air is being drained out of the room at a steady pace.

**Comment by BomberBro** [comment]
If you leave a shitty mechanic in long enough eventually people will think it's part of the game, see: explosions braining people with balls of paper

**Comment by SECBATON-GRIFFON** [comment]
ban this sick filth

**Comment by Kurfursten** [comment]
Lack of support
